Source/ISHRemote/Trisoft.ISHRemote/Objects/Public/IshSession.cs

```diff
@@ -141,13 +149,20 @@ private void CreateConnection()
             //prepare connection for authentication/authorization
             var connectionParameters = new InfoShareWcfConnectionParameters
             {
-                Credential = _ishSecurePassword == null ? new NetworkCredential() : new NetworkCredential(_ishUserName, SecureStringConversions.SecureStringToString(_ishSecurePassword)),
+                Credential = _ishSecurePassword == null ? null : new NetworkCredential(_ishUserName, SecureStringConversions.SecureStringToString(_ishSecurePassword)),
```

Comment:
When not providing credentials, it seems correct to retain credentials as `null` for the entire flow. It also makes a condition that decides about the binding type easier.